### DISCUSSION TITLE
[@types/node] LTS tag

### DISCUSSION THREAD

**Discussion by muuvmuuv** [DefinitelyTyped/DefinitelyTyped#61603] [Issues with a @types package]
Is it possible to have a LTS-Tag (`@types/node@lts`) for node? We don't need strict type version to bin version API comparison, so the LTS for types would work fine for us.

[a comment by an automated account is not shown]

**Comment by SimonSchick** [comment]
What would the value of that be? You should always match the type major+minor version to your nodejs version.
The `lts` tag may change at any moment when a release happens.

**Reply by muuvmuuv** [reply]
That in package managers it does not get reported as outdated since latest would be 18 and not 16 (current LTS). We already define fine-grained version through "engine" which is in our case always some LTS version. And we not really care about some LTS version differences (between types and engine)  since we are (sadly) not strictly check typing, and I guess there won't be any major issuer with it. So in our case it would always somehow match engine ↔ types.

**Reply by muuvmuuv** [reply]
Just some thought but if the tag "lts" would exist maybe we could also define LTS in engine. So it would become:

```json
{
	"engine": {
		"node": "lts"
	},
	"devDependencies": {
		"@types/node": "lts"
	}
}
```

**Reply by muuvmuuv** [reply]
We are not running an NodeJS API, so a strict Node Version isn't required, just LTS for libs and packages.

**Reply by SimonSchick** [reply]
This would be an issue for the types publisher maintainer.
I think your case is not very common, usually people lock major versions eg. `^18.0.0` rather than just taking whatever is available as lts, I'd actually strongly discourage that, you will have to update your types major version every year or so for the time being.

**Comment by peterblazejewicz** [comment]
I believe LTS is a convention, not a rigid thing (e.g. Angular I'm using at work, ships with 2 LTS version: last 2 major version, except of current one). IMO that would put additional burden on contributors (or maintainers)

**Reply by muuvmuuv** [reply]
Hmm yes got that point. It just tesses me to have one pkg that isnt on latest 😅 and most pin node to some lts
